src/theme/components/button.ts

```diff
@@ -1,20 +1,23 @@
 // Style object for base or default style
 const baseStyle = {
+  alignItems: "center",
   borderRadius: "sm",
-  lineHeight: "1.5",
   display: "flex",
   cursor: "pointer",
   color: "ui.white",
+  height: "10",
   justifyContent: "center",
+  lineHeight: "1.5",
   maxHeight: "2.5rem",
+  minHeight: { base: "44px", md: "auto" },
```

Comment:
Can the `"44px"` value be reusable somehow? Like a semantic token variable or a reusable object since the minHeight for button, select, and textinput all have the same.

Reply:
@EdwinGuzman Yes, let me see what I can do.